src/data/structure/Recommendation.java

```diff
@@ -0,0 +1,33 @@
+package data.structure;
+
+/**
+ * Represents a recommendation based on a UserProfile with its associated Preferences
+ *    (i.e. the user's preferences).  Contains the Preference that is most correlated
+ *      with the user's preferences as well as total correlation score.
+ * @author benjaminposerow
+ *
+ */
+public class Recommendation {
```

Comment:
Would there be value in implementing the equals() and hashcode() methods? I generally try to do so for POJOs such as this.